src/main/kotlin/calculator/type/Operator.kt

```diff
@@ -0,0 +1,17 @@
+package calculator.type
+
+enum class Operator(val symbol: String, val operationFunction: (a: Int, b: Int) -> Int) {
+    PLUS("+", { a, b -> a + b }),
+    MINUS("-", { a, b -> a - b }),
+    MULTIPLY("*", { a, b -> a * b }),
+    DIVIDE("/", { a, b -> a / b }),
+
+    NONE("", { a, b -> a })
+    ;
+
+    companion object {
+        fun of(symbol: String): Operator {
+            return values().find { it.symbol == symbol } ?: NONE
```

Comment:
operator가 아닌 symbol로 Operator를 찾으면 예외를 던지는 것이 조금 더 깔끔할 것 같아요. 🤔

예를들어 혹시라도 Operator로 `NONE` 이 반환되었을 때, 실수로 operationFunction을 실행하게되면, 잘못된 연산이 수행될 수도 있을 것 같아요. 👀

```suggestion
            return values().find { it.symbol == symbol } ?: throw IllegalArgumentException("존재하지않는 연산자 symbol입니다. symbol:$symbol")
```